src/ProjectTemplates/GrainImplementation.NetStandard.VB/Company.Grains.vbproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <!--/-:cnd -->
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Orleans.Core.Abstractions" Version="2.0.0-*" />
+  </ItemGroup>
```

Comment:
Since there is no runtime-codegen (yet at least), VB and F# projects require an extra C# project that references this just to run codegen. If the initial version of the templates don't have multi-project support (which is OK, we can add it later), at least a readme file in the project with this info would be highly beneficial.
These templates are not for power users, but as a way to get started and learn how to create an Orleans solution.